packages/client/components/EditorLinkChanger/EditorLinkChangerDraftjs.tsx

```diff
@@ -37,11 +51,49 @@ const EditorLinkChangerDraftjs = (props: Props) => {
   } = props
   useTaskChild('editor-link-changer')
   const handleSubmit = ({text, href}: {text: string; href: string}) => {
+    const keepSelection = true
```

Comment:
-1 please remove and manually switch to the true branch later

Reply:
> what you mean by branch

Here also.